connectors/yospace/README.md

```diff
@@ -0,0 +1,51 @@
+# THEOplayer Yospace Connector for Android
+
+The Yospace connector provides a Yospace integration for THEOplayer.
+
+## Installation
+
+1. Add the THEOplayer Maven repository to your project-level `settings.gradle` file:
+   ```groovy
+   dependencyResolutionManagement {
+       repositories {
+           google()
+           mavenCentral()
+           maven { url = uri("https://maven.theoplayer.com/releases/") }
+       }
+   }
+   ```
+2. Add THEOplayer and the Yospace Connector as a dependency in your module-level `build.gradle` file:
+   ```groovy
+   dependencies {
+       implementation "com.theoplayer.theoplayer-sdk-android:core:7.+"
+       implementation "com.theoplayer.android-connector:yospace:7.+"
+   }
+   ```
+
+## Usage
+
+First, follow [the getting started guide for the THEOplayer Android SDK][android-getting-started]
+to set up a `THEOplayerView` in your app.
+
+Then, create the `YospaceConnector`:
```

Comment:
We should also explain how to install the Yospace Ad Management SDK, since the connector depends on it. Or at least add a link to the Yospace docs where they explain this.